### PR TITLE
added stop button

### LINKED ISSUE #715: Stop button not visible in Scratch activity on small screens
Irresponsive screen and button to stop the activity is not visible.

WhatsApp Image 2020-03-19 at 10 56 41 PM

### PR DESCRIPTION
**Pull request by Swayamsvk**
fixed #715

### CODE REVIEW

**Comment by Swayamsvk** [comment]
scratch

**Comment by Swayamsvk** [comment]
@llaske I have added the stop button as you have mentioned please review this and tell me if anything else is necessary.

File: activities/Scratch.activity/css/activity.css [hunk 1]
```diff
@@ -40,3 +40,11 @@ body {
   overflow-x: hidden;
   overflow-y: hidden;
 }
+
+@media screen and (max-width: 900px) {
+	#stop-button{
+    position: absolute;
+		left: 85vw;
```
Comment (ashish0910) [inline activities/Scratch.activity/css/activity.css]: Please fix the indentation

**Comment by Swayamsvk** [comment]
@ashish0910 I have fixed indentation please review it and tell me if anything else is necessary.

**Comment by Swayamsvk** [comment]
@ashish0910 i have fixed indentation using tab please review.

**Comment by ashish0910** [comment]
@Swayamsvk Sorry but indentation is still not correct . You can see [here](https://github.com/Swayamsvk/sugarizer/blob/d25b2df07ecd4f2ff4f5d53be8f399e4a1a54825/activities/Scratch.activity/css/activity.css#L46) .

**Comment by Swayamsvk** [comment]
@ashish0910 I think I have to change the indentation of the whole part.Please tell me if I am right.

**Comment by Swayamsvk** [comment]
@ashish0910 actually I was having some problem in the indentation part of my editor using tab but now it seems to be fixed. please review.

**Comment by Swayamsvk** [comment]
@ashish0910 and @llaske please merge this PR.

**Comment by llaske** [comment]
Nice. Thanks.

**Pull request merged**